MediaBrowser.Model/Dlna/DeviceProfile.cs

```diff
@@ -31,104 +26,6 @@ public class DeviceProfile
         [XmlIgnore]
```

Comment:
Are those XmlIgnore attributes still needed? AFAIK only DLNA profiles are stored in XML.

Reply:
My plan was to move this code to a class called `DlnaDeviceProfile` that extends this class, so some of them are still needed unfortunately AFAIK. If there's an easy way to apply the attributes from the dlna plugin I'd prefer that, but idk.

Reply:
Might be possible with an override.

Reply:
Looks like it's possible, but there are some caveats associated, so I'll leave this as-is for now and play around with it later